packages/transition-backend/src/services/osm/OsmGetStreetsAroundPoint.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { circle as turfCircle } from '@turf/turf';
+import { DataGeojson } from 'chaire-lib-common/lib/tasks/dataImport/data/dataGeojson';
+import osmDownloader from 'chaire-lib-common/lib/utils/osm/OsmOverpassDownloader';
+import streetsXMLQuery from 'chaire-lib-common/lib/config/osm/overpassQueries/streets';
+import * as Status from 'chaire-lib-common/lib/utils/Status';
+
+export const getStreetsAroundPoint = async function (
+    aroundPoint: GeoJSON.Feature<GeoJSON.Point>,
+    radiusMeters: number
+): Promise<Status.Status<GeoJSON.Feature[]>> {
+    const circleGeojson = turfCircle(aroundPoint, radiusMeters, { units: 'meters' });
```

Comment:
What happen if radius is negative. 
(We probably need a test for that also)

Reply:
Added a validation for radius <= 0